runtimevar/runtimevar.go

```diff
@@ -479,6 +480,27 @@ func bytesDecode(b []byte, obj interface{}) error {
 	return nil
 }
 
+// DecryptDecode returns a decode function that can be passed to NewDecoder when
+// decoding an encrypted message (https://godoc.org/gocloud.dev/secrets).
+//
+// The decode function by default will decode the input into raw bytes. An
+// optional decoder can be passed in to do further decode operation based on the
+// decrypted message.
+func DecryptDecode(ctx context.Context, k *secrets.Keeper, post Decode) Decode {
+	return func(b []byte, obj interface{}) error {
+		decrypted, err := k.Decrypt(ctx, b)
+		if err != nil {
+			return err
+		}
+		if post == nil {
```

Comment:
Default to `bytesDecode` (or `BytesDecode` if you follow my suggestion above).

Reply:
Done.